### ISSUE TITLE
TITAGITSTANDARD does not store lock status in USB accessable memory

### ISSUE THREAD

**Issue by mschmitt-pf** [emsec/ChameleonMini#291]
Hi,
just found out that TITAGITSTANDARD does not store its lock status in USB accessable memory range (via UPLOAD  / DOWNLOAD command).

Im my case I want to use this to verify / emulate the lock state via USB access, as implemented for EM4233. 
In fact I first tried to use EM4233, but it just does not respond on e.g. "write to locked block". 

Hope you can fix this.

Thank you in advance.

**Comment by ceres-c** [comment]
Yes, EM4233 does not respond to locked block writes because that's exactly what a real tag does: no answer.

You can pretty easily edit the code in TITagitstandard.c to save in memory lock status as EM4233.c does
